guacamole-common/src/main/java/org/apache/guacamole/servlet/GuacamoleHTTPTunnelServlet.java

```diff
@@ -256,14 +259,18 @@ else if(query.startsWith(WRITE_PREFIX))
 
         // Catch any thrown guacamole exception and attempt to pass within the
         // HTTP response, logging each error appropriately.
-        catch (GuacamoleClientException e) {
-            logger.warn("HTTP tunnel request rejected: {}", e.getMessage());
-            sendError(response, e.getStatus(), e.getMessage());
-        }
         catch (GuacamoleException e) {
-            logger.error("HTTP tunnel request failed: {}", e.getMessage());
-            logger.debug("Internal error in HTTP tunnel.", e);
-            sendError(response, e.getStatus(), "Internal server error.");
+            if (e instanceof GuacamoleClientException) {
```

Comment:
There's no need to manually check `instanceof` - this is exactly what `catch` is for.

Reply:
Okay - I wasn't sure what the more accepted/elegant approach was.  Will rework it that way.

Reply:
Fixed.